docs/source/getting-started/model-registry/step2-load-registered-model.rst

```diff
@@ -0,0 +1,78 @@
+Registering a Model
+===================
+
+Now that we have registered a model, there are many ways to bring that model into memory. Typically,
+users reference a model by its name, which is a unique identifier determined when the model is 
+logged. However, you can also get the model via absolute paths to the artifact, run IDs combined 
+with the artifact path, etc.
+
+For completeness, here are various ways to load a model using MLflow's Fluent API:
+
+1. Absolute local path: `mlflow.sklearn.load_model("/Users/me/path/to/local/model")`
+2. Relative local path: `mlflow.sklearn.load_model("relative/path/to/local/model")`
+3. Run id: `mlflow.sklearn.load_model(f"runs:/{mlflow_run_id}/{relative_path_to_model}")`
+4. Model name + version: `mlflow.sklearn.load_model(f"models:/{model_name}/{model_version}")`
+5. Model name + stage alias: `mlflow.sklearn.load_model(f"models:/{model_name}/{model_stage_alias}")`
+
+Note that you can also leverage the MLflow client and raw REST API calls. However, for simplicity,
+we recommend using `mlflow.sklearn.load_model(f"models:/{model_name}/latest")` to get the latest 
+model version. 
+
+Let's see how we can load a model programmatically using information from the MLflow UI.
+
+Step 1: Open the MLflow UI
+--------------------------
+
+Let's first open up the UI. Note that you can get all this information programmatically as well, but
+the UI provides a single interface for all relevant information. 
+
+.. code-section::
+    .. code-block:: bash 
+        :name: start-mlflow-ui
+
+        mlflow ui
+
+Step 2: Get the Required Information 
+------------------------------------
+
+Next, let's inspect the UI and identify information that is relevant to loading our model. 
+
+
+.. figure:: ../../_static/images/tutorials/introductory/model-registry/mlflow_ui.png
+   :width: 1024px
+   :align: center
+   :alt: Model information from the mlflow ui.
+
+   Accessing model information from the model registry.
```

Comment:
I wondering if we should have users look at the registered model in the Model Registry UI instead. The UI here is tracking. It would be more direct to look at the Model Registry UI and see what registered models exist.

In fact, users don't need to open the UI to find the registered model name and version. The output of the previous step already shows the model name `'sk-learn-random-forest-reg-model'` and the model version `1`. The only reason one would open the UI in this tutorial, I would think is to set aliases on particular model versions using the UI.

Reply:
The MLflow UI defaults to tracking, so we need to show steps to getting the registry anyway. That adds a step and makes the tutorial more challenging. 

We are looking to enable users to access model information outside the context of a run.